tool/dev.dart

```diff
@@ -81,22 +81,22 @@ Future<Null> _serveExamples() {
   io.Process.start('pub', ['serve', '--port=9000'],
       workingDirectory: 'example');
 
-  return new Completer<Null>().future;
+  return Completer<Null>().future;
```

Comment:
Is the plan to update to dart_dev V3 in a followup?

Reply:
yeah, @evanweible-wf  has PR in progress https://github.com/Workiva/w_transport/pull/330 he started it in the summer. I asked him about it and we agreed that I'll just address `dartfmt`, `d1down` fixes and after that he'll update his PR to remove the rest of dart1.